app/code/Magento/Sales/Test/Mftf/Test/AdminCreateOrderWithConfigurableProductPressKeyEnterTest.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateOrderWithConfigurableProductPressKeyEnterTest">
+        <annotations>
+            <title value="Create Order in Admin with configurable product"/>
+            <stories value="AC-2040"/>
+            <description value="Create order with configurable product."/>
+            <features value="Sales"/>
+            <testCaseId value="AC-2040"/>
```

Comment:
It is duplicate testCaseId in Magento. I think you can leave it as empty like that test case

![image](https://user-images.githubusercontent.com/8234209/191873927-0777acd9-198a-4be5-a98d-7e17fc00bcf6.png)

Reply:
I removed it ^^

Reply:
I added new commit